src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsChecklist.tsx

```diff
@@ -1,6 +1,5 @@
-import { Box, Sans, StackableBorderBox } from "@artsy/palette"
+import { Box, color, Sans, StackableBorderBox } from "@artsy/palette"
```

Comment:
I think using color from Palette is a preferred way now, right?

Reply:
Yupp